packages/tonetwistplugs/PKGBUILD

```diff
@@ -0,0 +1,70 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Christopher Arndt <aur -at- chrisarndt -dot- de>
+
+_name=ToneTwistPlugs
+pkgname=${_name,,}
+pkgver=0.6
+pkgrel=1
+pkgdesc='A small collection of multi-format, cross-platform guitar pedal effect plugins'
+arch=(aarch64 x86_64)
+url="https://github.com/brummer10/$_name"
+license=(GPL2)
+groups=(clap-plugins lv2-plugins pro-audio vst-plugins vst3-plugins)
+depends=(cairo glibc gcc-libs libx11 libxcursor libxext libxrandr)
+makedepends=(dbus lv2)
+checkdepends=(lv2lint)
+optdepends=(
+  'jack: for using the stand-alone programs with JACK'
+  'clap-host: for loading the CLAP format plugins'
+  'lv2-host: for loading the LV2 format plugins'
+  'vst-host: for loading the VST2 format plugins'
+  'vst3-host: for loading the VST3 format plugins'
+)
+_dpf_commit=47e5f3ceeccf880119be0a539f9db892dc56fbef
+_pugl_commit=88dc250bfd4ca1ccd47801acf071ebf79e1450d7
+source=(
+  "$_name-$pkgver.tar.gz::https://github.com/brummer10/$_name/archive/refs/tags/v$pkgver.tar.gz"
+  "dpf-$_dpf_commit.tar.gz::https://github.com/DISTRHO/DPF/archive/$_dpf_commit.tar.gz"
+  "pugl-$_pugl_commit.tar.gz::https://github.com/DISTRHO/pugl/archive/$_pugl_commit.tar.gz"
+)
+sha256sums=('99190f9ddc03eb8586abdf8afcaa8ec8ecf81db7af3db7791949b5702ff3df33'
+            '5e7cf7fef94240420f345fcf68bf7c920e88256a11ea3ae264f55644a2fb9995'
+            'd83e958d8d699d60c2425eb51de7b38244b3f86e7a84ede83d6454b84cbf7ce5')
+_plugins=(
+  BoobTube
+  CollisionDrive
+  MetalTone
```

Comment:
Oh, just noticed this after approving the 2 other PRs. Doesn't it conflict with the other packages?

Reply:
No, two different implementations of the plugin with different LV2 URIs and installation dir.

Reply:
* `/usr/lib/lv2/collisiondrive.lv2` -> `urn:brummer10:collisiondrive`
* `/usr/lib/lv2/CollisionDrive.lv2` -> `https://github.com/brummer10/CollisionDrive`

Reply:
Indeed, I see no collision here 🥁